### PR TITLE
Add yUSDUSD, USDyUSD, COMPUSD, USDCOMP, YFIUSD, USDYFI, ALCXUSD, USDALCX, RUNEUSD, USDRUNE, ALPHAUSD, USDALPHA, MKRUSD, USDMKR, CRVUSD, USDCRV, RENUSD, USDREN, RGTUSD, USDRGT, NFTXUSD, and USDNFTX as price identifiers

### PR DESCRIPTION
**Pull request by pemulis**
Signed-off-by: John Shutt <john.d.shutt@gmail.com>

### CODE REVIEW

File: UMIPs/omnibus-prices-1.md [hunk 1]
```diff
@@ -0,0 +1,1169 @@
...
+- RULER/USD
+- USD/RULER
+
+The canonical identifiers should be `yUSDUSD`, `USDyUSD`, `COMPUSD`, `USDCOMP`, `YFIUSD`, `USDYFI`, `ALCXUSD`, `USDALCX`, `RUNEUSD`, `USDRUNE`, `ALPHAUSD`, `USDALPHA`, `MKRUSD`, `USDMKR`, `CRVUSD`, `USDCRV`, `RENUSD`, `USDREN`, `RGTUSD`, `USDRGT`, `NFTXUSD`, `USDNFTX`, `RULERUSD`, and `USDRULER`.
+
+# MOTIVATION
+
+1. What are the financial positions enabled by adding these price identifiers that do not already exist?
+
+- These price identifiers allow the use of the base currencies as collateral for minting synthetics or call options. See also [UMIP-X]().
```
Comment (ChandlerDeKock) [inline UMIPs/omnibus-prices-1.md:47]: should add this when you get a chance

File: UMIPs/omnibus-prices-1.md [hunk 2]
```diff
@@ -0,0 +1,1169 @@
...
+## IMPLEMENTATION
+
+Voters should query for the price of YFI/USD at the price request timestamp on Kraken and Coinbase Pro. Recommended endpoints are provided in the markets and data sources section.
+
+1. When using the recommended endpoints for Kraken and Coinbase Pro, voters should use the open price of the 1 minute OHLC period that the timestamp falls in.
+2. Also query YFI/ETH Price from SushiSwap using 1 minute TWAP (time weighted average price).
+3. Query the ETH/USD Price as per UMIP-6.
+4. Multiply the YFI/ETH price by the ETH/USD price and round to 6 decimals to get the YFI/USD price.
+5. The median of the Kraken, Coinbase Pro, and SushiSwap results should be taken.
+6. The median from step 2 should be rounded to six decimals to determine the YFIUSD price.
```
Comment (Reinis-FRP) [inline UMIPs/omnibus-prices-1.md:392]: Step 2 has only one YFI/ETH price, so median is not applicable
Is it meant median from step 5?

File: UMIPs/omnibus-prices-1.md [hunk 3]
```diff
@@ -0,0 +1,1169 @@
...
+**Is your collateral currency already approved to be used by UMA financial contracts?:** In progress.
+
+## IMPLEMENTATION
+
+Voters should query for the price of YFI/USD at the price request timestamp on Kraken and Coinbase Pro. Recommended endpoints are provided in the markets and data sources section.
+
+1. When using the recommended endpoints for Kraken and Coinbase Pro, voters should use the open price of the 1 minute OHLC period that the timestamp falls in.
+2. Also query YFI/ETH Price from SushiSwap using 1 minute TWAP (time weighted average price).
+3. Query the ETH/USD Price as per UMIP-6.
+4. Multiply the YFI/ETH price by the ETH/USD price and round to 6 decimals to get the YFI/USD price.
```
Comment (Reinis-FRP) [inline UMIPs/omnibus-prices-1.md:390]: Why round it here if median is rounded afterwards?

File: UMIPs/omnibus-prices-1.md [hunk 4]
```diff
@@ -0,0 +1,1169 @@
...
+
+## IMPLEMENTATION
+
+```
+1. Query yUSD/ETH Price from SushiSwap using 1 hour TWAP.
+2. Query the ETH/USD Price as per UMIP-6.
+3. Multiply the yUSD/ETH price by the ETH/USD price and round to 6 decimals to get the yUSD/USD price.
+4. (for USD/yUSD) Take the inverse of the result of step 3 (1/ yUSD/USD) to get the USD/yUSD price.
+```
+
```
Comment (mhairimca) [inline UMIPs/omnibus-prices-1.md:502]: Similar problem with a twap price being multiplied by a spot price in a time of volitility.

File: UMIPs/omnibus-prices-1.md [hunk 5]
```diff
@@ -0,0 +1,1169 @@
...
+**Is your collateral currency already approved to be used by UMA financial contracts?:** In progress.
+
+## IMPLEMENTATION
+
+```
+1. Query RGT/ETH Price from SushiSwap using 1 hour TWAP.
+2. Query the ETH/USD Price as per UMIP-6.
+3. Multiply the RGT/ETH price by the ETH/USD price and round to 6 decimals to get the RGT/USD price.
+4. (for USD/RGT) Take the inverse of the result of step 3 (1/ RGT/USD) to get the USD/RGT price.
+```
```
Comment (mhairimca) [inline UMIPs/omnibus-prices-1.md:924]: This also has the problem of a twap and a spot price being multiplied

File: UMIPs/omnibus-prices-1.md [hunk 6]
```diff
@@ -0,0 +1,1169 @@
...
+**Is your collateral currency already approved to be used by UMA financial contracts?:** In progress.
+
+## IMPLEMENTATION
+
+```
+1. Query NFTX/ETH Price from SushiSwap using 1 hour TWAP.
+2. Query the ETH/USD Price as per UMIP-6.
+3. Multiply the NFTX/ETH price by the ETH/USD price and round to 6 decimals to get the NFTX/USD price.
+4. (for USD/NFTX) Take the inverse of the result of step 3 (1/ NFTX/USD) to get the USD/NFTX price.
+```
```
Comment (mhairimca) [inline UMIPs/omnibus-prices-1.md:1036]: Similar problem - TWAP x Spot price.

File: UMIPs/omnibus-prices-1.md [hunk 7]
```diff
@@ -0,0 +1,1169 @@
...
+
+**Pricing interval**
+- Every block
+
+**Input processing**
+- None.
+
+**Result processing**
+- See rounding rules in `Technical Specification`.
+
```
Comment (mhairimca) [inline UMIPs/omnibus-prices-1.md:1163]: Similar problem, TWAP x Spot price

File: UMIPs/omnibus-prices-1.md [hunk 8]
```diff
@@ -0,0 +1,1169 @@
...
+**Does the value of this collateral currency match the standalone value of the listed quote currency?:** Yes.
+
+**Is your collateral currency already approved to be used by UMA financial contracts?:** In progress.
+
+## IMPLEMENTATION
+
+```
+1. Query yUSD/ETH Price from SushiSwap using 1 hour TWAP.
+2. Query the ETH/USD Price as per UMIP-6.
+3. Multiply the yUSD/ETH price by the ETH/USD price and round to 6 decimals to get the yUSD/USD price.
```
[the remark this answers is not in the document]
Comment (pemulis) [reply UMIPs/omnibus-prices-1.md]: Dropped the TWAP to 15 minutes across the board for these situations; good catch.

File: UMIPs/omnibus-prices-1.md [hunk 9]
```diff
@@ -0,0 +1,1169 @@
...
+4. (for USD/COMP) Take the inverse of the result of step 3 (1/ COMP/USD) to get the USD/COMP price.
+
+For both implementations, voters should determine whether the returned price differs from broad market consensus. This is meant to provide flexibility in any unforeseen circumstances as voters are responsible for defining broad market consensus.
+
+# YFI
+
+## MARKETS & DATA SOURCES
+**Required questions**
+
+Markets: SushiSwap, Kraken, Coinbase Pro
```
[the remark this answers is not in the document]
Comment (pemulis) [reply UMIPs/omnibus-prices-1.md:283]: No particular reason for Kraken over Binance, so I can make that switch. SushiSwap has lower volume, but the best liquidity on the +2/-2 charts. Can replace SushiSwap with OKEx for simplicity, though.

**Review by smb2796 — APPROVED** [review]
Looks like all outstanding comments were addressed. Going to approve and move to last call.

File: UMIPs/omnibus-prices-1.md [hunk 10]
```diff
@@ -0,0 +1,1160 @@
...
+
+**Scaling Decimals:** 18 (1e18)
+
+**Rounding:** Round to nearest 6 decimal places (seventh decimal place digit >= 5 rounds up and < 5 rounds down)
+
+**Does the value of this collateral currency match the standalone value of the listed quote currency?:** Yes.
+
+**Is your collateral currency already approved to be used by UMA financial contracts?:** Yes.
+
+### USD/yUSD
```
Comment (gitofdeepanshu) [inline UMIPs/omnibus-prices-1.md:258]: [TYPO] :Change to **USD/RAI**

**Pull request merged**